src/platform/nrfconnect/CHIPDevicePlatformConfig.h

```diff
@@ -92,6 +92,8 @@
 
 #define CHIP_DEVICE_CONFIG_ENABLE_THREAD CONFIG_NET_L2_OPENTHREAD
 
+#define CHIP_DEVICE_CONFIG_ENABLE_WIFI CONFIG_NRF_700X
```

Comment:
I think you can use CONFIG_CHIP_WIFI here which can be more understandable than NRF_700X.

Reply:
I oppose to that as long as CONFIG_CHIP_WIFI is part of Kconfig.features. Note that Kconfig.features is an optional file which we include in examples to help enable a set of different features, but only `config/nrfconnect/chip-module/Kconfig` is mandatory file of the CHIP module.

Reply:
Ok, you're right, it makes sense at this stage, and we can change it after moving WIFI configs to the main Kconfig.